api/src/main/java/org/pmiops/workbench/ras/RasLinkConstants.java

```diff
@@ -27,14 +27,11 @@ private RasLinkConstants() {}
   // The login.gov's PREFERRED_USERNAME field from RAS UserInfo endpoint
   static final String PREFERRED_USERNAME_FIELD_NAME = "preferred_username";
 
-  // The login.gov's email field which we choose as login.gov username.
-  static final String EMAIl_FIELD_NAME = "email";
-
   // The identifier that indicate users login using login.gov account.
   static final String LOGIN_GOV_IDENTIFIER_LOWER_CASE = "@login.gov";
 
   // The string in ACR claim that we can use to identify user IAL status.
-  static final String ACR_CLAIM_IAL_2_IDENTIFIER = "https://stsstg.nih.gov/assurance/ial/2";
+  static final String ACR_CLAIM_IAL_2_IDENTIFIER = "/assurance/ial/2";
```

Comment:
Is this the exact value? Or are we just matching a suffix now?

Reply:
It is `https://stsstg.nih.gov/assurance/ial/2 https://stsstg.nih.gov/assurance/aal2`. Explained in previous comment, matching might be engough?. Do you think we should do the exact match or suffix match? 
I saw few different RAS urls prefix, so little bit concerns if the prefix can alwasy match, hence choose this suffiix way...

Reply:
Without having documentation from RAS on this, my default preference would be full match - partial match could potentially be exploited depending on where this value is coming from.

However, if you feel the RAS API is unstable here and we don't have good documentation from them yet, I'm fine with the partial suffix match for now (but I would probably have a ticket to come back and update this ahead of launch).

Reply:
I do have their documents and available in google doc. But they ask to delete from Google. 
It is in their box account: https://app.box.com/file/751989029656?s=wnjs4a0nqavhjwy3wdu5nv1vwlkx7dmk
Screen shot on IAL and AAL intro in this claim:
https://screenshot.googleplex.com/A2hx9t42Y5DeKRd